pages/hub/apps/sdk/data-hooks/useWaveLeaderBoard.mdx

```diff
@@ -0,0 +1,51 @@
+import { Callout } from 'components'
+
+Hook is used to get the [Wave Total Leaderboard](/hub/apps/guides/azuro-waves#fetch-total-leaderboard) data or [Wave Leaderboard for Specific Period](/hub/apps/guides/azuro-waves#fetch-leaderboard-for-specific-period).
+
+<Callout type="info">
+  This hook is wrapper around [getWaveLevels](/hub/apps/sdk/utils/getWaveLevels) function.
```

Comment:
wrong hook name and link

Reply:
thanks
0f86ef07421d7347f86c7aca7a532616ae866ca9